### PR TITLE
Update cli output on README

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

**Comment by ssbarnea** [comment]
recheck

File: docs/build [hunk 1]
```diff
@@ -0,0 +1 @@
+../docs/docsite/_build
```
Comment (webknjaz) [inline docs/build:1]: this is weird. why is this not `./docsite/_build` then? also, why can't this be fixed on Zuul side? I'd rather not have symlinks in this repo

Comment (ssbarnea) [reply docs/build]: I can change the location using a parameter for jobs defined inside the repo. For the moment I am only trying to see if new location is enough. 

If I can avoid the symlink, I will do it.

File: README.rst [hunk 2]
```diff
@@ -86,36 +86,43 @@ The following is the output from ``ansible-lint --help``, providing an overview
...
       -q                    quieter, although not silent output
       -p                    parseable output in the format of pep8
       --parseable-severity  parseable output including severity of rule
       -r RULESDIR           specify one or more rules directories using one or
                             more -r arguments. Any -r flags override the default
-                            rules in ['/path/to/ansible-
-                            lint/lib/ansiblelint/rules'], unless -R is also used.
-      -R                    Use default rules ['/path/to/ansible-
-                            lint/lib/ansiblelint/rules'] in addition to any extra
+                            rules in /Users/ssbarnea/c/os/ansible-
```
Comment (danielmellado) [inline README.rst:106]: I'd leave this as it was before ;)

File: README.rst [hunk 3]
```diff
@@ -86,36 +86,43 @@ The following is the output from ``ansible-lint --help``, providing an overview
...
+                            lint/lib/ansiblelint/rules, unless -R is also used.
+      -R                    Use default rules in /Users/ssbarnea/c/os/ansible-
```
Comment (danielmellado) [inline README.rst:108]: Same here, please keep the default ['path/to/ansible-lint...

**Comment by webknjaz** [comment]
> Creates a relative symlink to the standard location of sphinx build output in order to allow Zuul CI to find the built docs.

@ssbarnea you copy-paste foo has failed you :)

**Pull request merged**